src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -218,6 +218,120 @@ public static String composeRpcMethodHeaderSampleCode(
         composeUnaryRpcMethodSampleCode(method, clientType, arguments, resourceNames));
   }
 
+  public static String composeRpcDefaultMethodHeaderSampleCode(
+      Method method,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes) {
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+    // Assign method's request variable with the default value.
+    VariableExpr requestVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("request").setType(method.inputType()).build());
+    Message requestMessage = messageTypes.get(method.inputType().reference().simpleName());
+    Preconditions.checkNotNull(requestMessage);
+    Expr requestBuilderExpr =
+        DefaultValueComposer.createSimpleMessageBuilderExpr(
+            requestMessage, resourceNames, messageTypes);
```

Comment:
@miraleung A requestion regarding to building request variable. I copy the way you build request in `ClientTest`. It introduce the difference with `gapic-generator`, for example in googleapis java-asset::searchAllResource
Our [code](https://github.com/googleapis/gapic-generator-java/pull/594/files#diff-aed1368c14061ec77aca6c1b2731f60e07c8686baddac4f2b0cc72a9452b3487R691-R707):
```
*   SearchAllResourcesRequest request =
   *       SearchAllResourcesRequest.newBuilder()
   *           .setScope("scope109264468")
   *           .setQuery("query107944136")
   *           .addAllAssetTypes(new ArrayList<String>())
   *           .setPageSize(883849137)
   *           .setPageToken("pageToken873572522")
   *           .setOrderBy("orderBy-1207110587")
   *           .build();
```
vs. existing gapic-generator
```
   *   String scope = "";
   *   SearchAllResourcesRequest request = SearchAllResourcesRequest.newBuilder()
   *     .setScope(scope)
   *     .build();
```

Another approach here is to use the first method signatures to construct the request variable.
```
List<MethodArgument> arguments = !method.methodSignatures().isEmpty() ? method.methodSignatures().get(0) : Collections.emptyList();
// create list of argument varaible exprs
// create list of argument's default value exprs
// create list of assignment expr for arguments with default values
// Build request with by `String.format("set%s", JavaStyle.toUpperCamelCase(arg.name()))`
```
This approach can match the `gapic-generator` and reduce other field to set.

As summary, there are 2 approaches to build up request variable:
1. Use `createSimpleMessageBuilderExpr`
2. Use first method arguments's default values

Which one you prefer? Thanks.

Reply:
Our new one looks fine, let's stick with that.

Reply:
Cool. Thanks for confirm.